### PR TITLE
Added eslint import/no-duplicates

### PR DESCRIPTION
**Pull request by jguddas**
- chore: added eslint import/no-duplicates
- chore: unified duplicate import

Closes # 

## 📝 Description

> Add a brief description

## ⛳️ Current behavior (updates)

> Please describe the current behavior that you are modifying

## 🚀 New behavior

> Please describe the behavior or changes this PR adds

### CODE REVIEW

**Comment by jguddas** [comment]
@gauravkrp what do you think?

**Review by gauravkrp — APPROVED** [review]
@jguddas looks good!

File: packages/react/src/tooltip/tooltip.tsx [hunk 1]
```diff
@@ -4,11 +4,10 @@ import React, {useEffect, useRef, useState} from "react";
 
 import withDefaults from "../utils/with-defaults";
 import useClickAway from "../use-click-away";
-import {Placement} from "../utils/prop-types";
-import {TriggerTypes} from "../utils/prop-types";
+import {Placement, TriggerTypes} from "../utils/prop-types";
+import {CSS} from "../theme/stitches.config";
```
Comment (tianenpang) [inline packages/react/src/tooltip/tooltip.tsx:8]: Hey @jguddas duplicate `CSS` type imported here.

**Review by tianenpang — CHANGES_REQUESTED** [review]

**Review by tianenpang — APPROVED** [review]
Hi @jrgarciadev LGTM, ready to merge. Thanks @jguddas 🚀

**Comment by jrgarciadev** [comment]
Huge thanks @jguddas! 🚀

**Pull request merged**